assets/blocks/quiz/quiz.editor.scss

```diff
@@ -72,6 +72,12 @@ $gray-400: #ccc;
 	}
 
 	/* Settings */
+	&__settings-link {
+		display: flex;
+		margin: 0 auto;
+		text-decoration: underline;
+	}
+
```

Comment:
I would also remove this border when clicked: 
![image](https://user-images.githubusercontent.com/7208249/148513253-2bff1bb9-1fb8-4772-962c-7c335c1a14d3.png)

You can do that by adding following 

```
	&__settings-link:focus:not(:disabled) {
		box-shadow: none;
	}
```

Reply:
Great suggestion, thanks! Updated in cf3a95223564b9085b259f2869c29c3147ba1c54.